sycl/test-e2e/bindless_images/cubemap/cubemap_unsampled.cpp

```diff
@@ -1,6 +1,6 @@
 // REQUIRES: cuda,aspect-ext_oneapi_cubemap
 
-// RUN: %clangxx -fsycl -fsycl-targets=%{sycl_triple} %s -o %t.out
+// RUN: %{build} %s -o %t.out
```

Comment:
```suggestion
// RUN: %{build} -o %t.out
```

I've run the current patch locally and all tests fail. It seems like `%{build}` already contains `%s`.

Reply:
Yep, I missed that. Will add it shortly